scss/_utilities.scss

```diff
@@ -98,55 +98,64 @@ $utilities: map-merge(
     // scss-docs-start utils-borders
     "border": (
       property: border,
+      local-vars: (
```

Comment:
Need to discuss about this var as it is never used for me

Reply:
This is the default value for opacity in http://localhost:9001/docs/5.1/utilities/borders/.